src/components/elements/ClickToCopy.tsx

```diff
@@ -26,9 +34,26 @@ export const ClickToCopyId: React.FC<Props> = ({
           sx={{ fontSize: 'inherit' }}
           size='small'
         >
-          <ContentCopyIcon fontSize='inherit' />
+          <CopyIcon fontSize='inherit' />
         </IconButton>
       )}
     </Typography>
   );
 };
+
+export const ClickToCopyChip: React.FC<{ value: string } & ChipProps> = ({
+  value,
+  sx,
+  ...props
+}) => {
+  return (
+    <Chip
```

Comment:
Just checking but is does this render a button element? It probably does but if not maybe add aria `role="button"`

Reply:
good call - looks like mui already adds the role